.github/workflows/compressed-size.yml

```diff
@@ -46,3 +46,4 @@ jobs:
           # The sub-match below will be replaced by asterisks.
           # The length of 20 corresponds to webpack's `output.hashDigestLength`.
           strip-hash: "([a-f0-9]{20})(?:\\.min)?\\.(?:css|js)$"
+          minimum-change-threshold: 204800 # 200KB
```

Comment:
I realize this is in the IB but 200KB is 200,000 bytes :)
<img width="466" alt="image" src="https://github.com/user-attachments/assets/c705fdfd-6dd6-41db-8837-cb884bd4e6f7" />